src/resolvers/index.ts

```diff
@@ -41,8 +47,10 @@ export const ObjectIdScalar = new GraphQLScalarType({
 
 type PortalUserContext = {
   db: typeof MongoClient
-  user: PortalUser
-  keystoneUrl: string
```

Comment:
realized that since we use data sources now, we do not need to access the keystone URL in the resolvers 🎉